src/examples/forms/toggle/Validation.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Toggle, Message } from '@zendeskgarden/react-forms';
+
+const StyledField = styled(Field)`
+  margin-top: ${p => p.theme.space.xs};
+`;
+
+const Example = () => {
+  return (
+    <Row justifyContent="center">
+      <Col size="auto" textAlign="start">
```

Comment:
Doesn't look like `textAlign` is doing anything here.

Reply:
Whoops, removed the unnecessary `textAlign` 👍 .